### PR TITLE
MNT: Re-rendered with conda-smithy 2.0.1

### PR DESCRIPTION
**Pull request by ericmjl**

### CODE REVIEW

**Comment by ericmjl** [comment]
ping @springcoil; re-rendering so that there can be a py36 build.

File: .travis.yml [hunk 1]
```diff
@@ -4,34 +4,46 @@
...
-osx_image: beta-xcode6.1
+osx_image: xcode6.4
 
 env:
   matrix:
     
     - CONDA_PY=27
-    - CONDA_PY=34
     - CONDA_PY=35
-    - CONDA_PY=36
```
Comment (ocefpaf) [inline .travis.yml]: Something is not OK with this re-rendering. Let me try it locally..

**Comment by ocefpaf** [comment]
It is skipped in https://github.com/conda-forge/pymc3-feedstock/blob/master/recipe/meta.yaml#L16

But I bet some of those dependencies are not available for Python 3.6 yet.

**Comment by ericmjl** [comment]
@ocefpaf: I'm going to try again. Skipping py27, added py36.

File: .travis.yml [hunk 2]
```diff
@@ -4,13 +4,11 @@
...
 
 os: osx
-osx_image: beta-xcode6.1
+osx_image: xcode6.4
 
 env:
   matrix:
     
-    - CONDA_PY=27
-    - CONDA_PY=34
```
Comment (ocefpaf) [inline .travis.yml]: That looks correct now. (conda-forge dropped PY34 and the pre-selector you added dropped PY27.)

**Comment by twiecki** [comment]
Can we merge this?

**Comment by ocefpaf** [comment]
> Can we merge this?

All the CIs failed... I would not recommend. Let me investigate this. Hold on...

**Comment by ocefpaf** [comment]
It seems that `theano` is missing for Python 3.6.

**Comment by ocefpaf** [comment]
See https://github.com/conda-forge/theano-feedstock/issues/6

**Comment by ericmjl** [comment]
Not sure what the error is now? It shows up a few times in AppVeyor.

```
conda.exceptions.UnsatisfiableError: The following specifications were found to be in conflict:
  - pymc3 3.0 py36_0
  - python 3.6*
Use "conda info <package>" to see the dependencies for each package.
```

**Comment by ocefpaf** [comment]
That is probably because `theano` is not available yet: https://ci.appveyor.com/project/conda-forge/theano-feedstock/build/1.0.10 I will restart Appveyor to put it in the queue after `theano`.

**Comment by twiecki** [comment]
As https://github.com/conda-forge/theano-feedstock/pull/6 is merged, can we go ahead on this one?

**Comment by ocefpaf** [comment]
> As conda-forge/theano-feedstock#6 is merged, can we go ahead on this one?

The answer is: probably yes. But I restarted Travis-CI and we can wait for the green to be sure.

**Comment by ericmjl** [comment]
All checks pass! Can I merge, @ocefpaf? I'd love to do the honours for PyMC3 :smile:

**Comment by ocefpaf** [comment]
> All checks pass! Can I merge, @ocefpaf? I'd love to do the honours for PyMC3 :smile: 

Yes please!

**Comment by ericmjl** [comment]
@ocefpaf @twiecki all done! :smile:

**Comment by twiecki** [comment]
Thanks @ericmjl and @ocefpaf!

**Comment by springcoil** [comment]
Excellent work guys :)

**Pull request merged**